pkg/controller/rds/dbinstance/setup.go

```diff
@@ -374,9 +387,15 @@ func (e *custom) isUpToDate(cr *svcapitypes.DBInstance, out *svcsdk.DescribeDBIn
 
 	}
 
+	vpcSGsChanged := !areVPCSecurityGroupIDsUpToDate(cr, db)
+
+	dbParameterGroupChanged := !isDBParameterGroupNameUpToDate(cr, db)
```

Comment:
Maybe don't invert it twice but name it `*isUpToDate`

Reply:
Wanted to stay close within the already existing "changed"-logic inside `isUpToDate()`:
https://github.com/crossplane-contrib/provider-aws/blob/dd12a90d3a3d6d2d935566161471600478698b3a/pkg/controller/rds/dbinstance/setup.go#L412

Reply:
Alright, then lets keep it like that.